resources/articles/oauth/register.md

```diff
@@ -0,0 +1,75 @@
+:title POST /oauth/introspect
+
+:aside
+
+## See also
+
+* [POST /oauth/token](/oauth/token/)
+* [POST /oauth/ro](/oauth/ro/)
+* [GET /oauth/userinfo](/oauth/userinfo/)
+* [GET /oauth/jwks](/oauth/jwks/)
+* [GET /oauth/authorize](/oauth/authorize/)
+* [Server to Server Auth](/s2s-auth/)
+* [Token introspection](/token-introspection/)
+
+:body
+
+The `/oauth/register` endpoint is used to register a new client under a merchant. The use case
```

Comment:
"The use case for this endpoint is mainly dynamically registered clients for s2s auth." -> "The use case for this is to allow merchants to dynamically register clients for [server to server auth]." <- add link to (`/s2s-auth/`).